mappings/net/minecraft/block/entity/BeaconBlockEntity.mapping

```diff
@@ -36,6 +36,8 @@ CLASS net/minecraft/unmapped/C_wwadquuj net/minecraft/block/entity/BeaconBlockEn
 		ARG 1 x
 		ARG 2 y
 		ARG 3 z
+	METHOD m_jlfysuud getEffect (Lnet/minecraft/unmapped/C_jaqasomh;)Lnet/minecraft/unmapped/C_jaqasomh;
```

Comment:
i still have a bad feeling about getting rid of the `orNull`; it just feels reasonable here, it's like, a standard too;
please reconsider it before merge

Reply:
Can you give an example of a similar method with the `OrNull` suffix?
I've been comparing it to `Map#get`, but there could be a better comparison.

Reply:
Actually maybe part of the issue is `get` might be the wrong verb here.
It checks if the beacon has the effect, returns it if so, or null otherwise.
It's like a weird `contains` method that returns arg/null instead of true/false.

Reply:
This method seems to be replacing what mojmap called `getValidEffectById`, I think valid is important.
I also think `effect` in the name is redundant given the parameter.
How about just `validate`?

Reply:
after looking through the source, I agree with @supersaiyansubtlety on `validate` being the best name
I'm going to add javadoc too

Reply:
@EnnuiL please take a look at the new name!

Reply:
Looks good to me; I'll double-check on Enigma soon, since uh, `getContainerOrNull` still lost the `OrNull` and i remember that it was there for good reasons

Reply:
it's annotated as `@Nullable` which imo makes the suffix redundant always

Reply:
it's annotated as `@Nullable` which imo makes the suffix redundant always

Reply:
fair enough